dGame/dGameMessages/GameMessages.cpp

```diff
@@ -5135,12 +5141,12 @@ void GameMessages::HandleModularBuildConvertModel(RakNet::BitStream* inStream, E
 
 	item->Disassemble(TEMP_MODELS);
 
-	item->SetCount(item->GetCount() - 1, false, false, true, eLootSourceType::LOOT_SOURCE_QUICKBUILD);
+	item->SetCount(item->GetCount() - 1, false, false, true, eLootSourceType::QUICKBUILD);
 }
 
 void GameMessages::HandleSetFlag(RakNet::BitStream* inStream, Entity* entity) {
 	bool bFlag{};
-	int iFlagID{};
+	uint32_t iFlagID{};
```

Comment:
I will note, flags are signed ints, and not unsigned ints.

Reply:
I thought the underlying type for int was uint32_t, my mistake